.github/workflows/build.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         if: env.publish == 'true'
         env:
           NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+          publish: ${{ env.publish }}
```

Comment:
Declares this env variable so that the GHA linter stops warning that publish might be invalid. Shouldn't have any effect on how this step runs.